src/main/java/frc/robot/commands/shooter/AdvancedSpinningShot.java

```diff
@@ -44,12 +39,11 @@ public void execute() {
     @Override
     public void end(boolean interrupted) {
         shooter.stopShooter();
-        timer.stop();
     }
 
     @Override
     public boolean isFinished() {
-        return timer.hasElapsed(Constants.ADVANCED_SPINNING_SHOT_TIMEOUT) || alignable.get() == null;
+        return alignable.get() == null;
```

Comment:
Again, this runs the risk of missing the termination condition, and likely needs a timeout.

Reply:
added deadline command